WordPress/Classes/Utility/WPImmuTableRows.swift

```diff
@@ -451,3 +451,37 @@ class ExpandableRow: ImmuTableRow {
         cell.urlCallback = onLinkTap
     }
 }
+
+struct SwiftUIRow<T: View>: ImmuTableRow {
```

Comment:
this new type is no longer used by this PR, but since I still find it generally useful I was thinking to leave it here if that's also fine with you?

Reply:
I'm thinking we can drop the code so there isn't any dead code but keep the commit so it can be referenced in the future if necessary.